spec/features/user_auth_spec.rb

```diff
@@ -41,18 +42,18 @@
       expect(page).to have_content('You are not authorized to access this page.')
     end
 
-    xit 'prevents a user from viewing the new category page' do
+    it 'prevents a user from viewing the new category page' do
       visit new_admin_category_path
       expect(page).to have_content('You are not authorized to access this page.')
     end
 
-    xit 'prevents a user from viewing a category page' do
+    it 'prevents a user from viewing a category page' do
       category = FactoryGirl.create(:category)
       visit new_admin_category_path(category)
       expect(page).to have_content('You are not authorized to access this page.')
     end
 
-    xit 'prevents a user from viewing the orders dashboard' do
+    it 'prevents a user from viewing the orders dashboard' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.